processing/data_collection/gazette/spiders/pr_curitiba.py

```diff
@@ -0,0 +1,119 @@
+        month = response.meta["month"]
+        #The first page of pagination cannot be accessed by page number
+        yield scrapy.FormRequest.from_response(
+            response,
+            formdata={
+                '__EVENTTARGET': 'ctl00$cphMasterPrincipal$TabContainer1',
+                'ctl00_cphMasterPrincipal_TabContalegacyDealPooliner1_ClientState' : '{{"ActiveTabIndex":{},"TabState":[true,true,true,true,true,true,true,true,true,true,true,true]}}'.format(month),
+                '__EVENTARGUMENT': 'activeTabChanged:{}'.format(month),
+            },
+            callback=self.parse_page,
+        )
+        for page_number in range(2, page_count + 1):
+            yield scrapy.FormRequest.from_response(
+                response,
+                formdata={
+                    '__EVENTARGUMENT' : 'Page${}'.format(page_number),
+                    '__EVENTTARGET' : 'ctl00$cphMasterPrincipal$gdvGrid2'
+                },
+                callback=self.parse_page,
+            )
+    def parse_page(self, response):
+        row = response.css(".grid_Row")
+        numbers = row.css("td:nth-child(1) span ::text").extract()
+        pdf_dates = row.css("td:nth-child(2) span ::text").extract()
+        ids = row.css("td:nth-child(3) a ::attr(data-teste)").extract()
+        for i in range(len(numbers)):
+            number = numbers[i]
+            pdf_date = pdf_dates[i]
+            id = ids[i]
+            parsed_date = parse(f'{pdf_date}', languages=['pt']).date()
+            if id == '0':
+                yield scrapy.FormRequest.from_response(
+                    response,
+                    headers = {'user-agent': 'Mozilla/5.0'},
+                    formdata = {
+                        '__LASTFOCUS': '',
+                        '__EVENTTARGET': 'ctl00$cphMasterPrincipal$gdvGrid2$ctl{num:02d}$lnkVisualizar'.format(num=(i+3)),
+                        '__EVENTARGUMENT': '',
+                        '__ASYNCPOST': 'true'
+                    },
+                    callback=self.scrap_not_extra_edition,
+                    meta={"parsed_date": parsed_date}
+                )
+            else:
+                yield Gazette(
+                    date = parsed_date,
+                    file_urls=["http://legisladocexterno.curitiba.pr.gov.br/DiarioSuplementoConsultaExterna_Download.aspx?id={}".format(id)],
+                    is_extra_edition= True,
+                    municipality_id=self.MUNICIPALITY_ID,
+                    power='executive_legislature',
+                    scraped_at=dt.datetime.utcnow()
+                )
+
+    def scrap_not_extra_edition(self, response):
+        parsed_date = response.meta['parsed_date']
+        id = re.findall(r'Id=(\d+)', response.text )[0]
+        return Gazette(
+            date = parsed_date,
+            file_urls=["http://legisladocexterno.curitiba.pr.gov.br/DiarioConsultaExterna_Download.aspx?id={}".format(id)],
```

Comment:
`id` is a built-in function in Python (https://docs.python.org/3.6/library/functions.html#id). Don't use it as a variable name. Give a better name like `pdf_id` to avoid problems.